extensions/amazon-aws/src/glue-jobs.tsx

```diff
@@ -0,0 +1,56 @@
+import { GlueClient, ListJobsCommand } from "@aws-sdk/client-glue";
+import { Action, ActionPanel, Icon, List } from "@raycast/api";
+import { useCachedPromise } from "@raycast/utils";
+import AWSProfileDropdown from "./components/searchbar/aws-profile-dropdown";
+import { isReadyToFetch, resourceToConsoleLink } from "./util";
+import { AwsAction } from "./components/common/action";
+
+export default function GlueJobs() {
+  const { data: glueJobs, error, isLoading, revalidate } = useCachedPromise(fetchJobs);
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder="Filter Jobs by name..."
+      searchBarAccessory={<AWSProfileDropdown onProfileSelected={revalidate} />}
+    >
+      {error ? (
+        <List.EmptyView title={error.name} description={error.message} icon={Icon.Warning} />
+      ) : (
+        glueJobs?.map((job) => <GlueJob key={job} job={job} />)
+      )}
+    </List>
+  );
+}
+
+function GlueJob({ job: glueJobName }: { job: string }) {
+  return (
+    <List.Item
+      key={glueJobName}
+      icon={"aws-icons/glue.png"}
```

Comment:
nit (non-blocking): can you please add a rounded rectangle mask to the img? Here is an [example](https://github.com/raycast/extensions/blob/main/extensions/amazon-aws/src/console.tsx#L22).